v2/cn/ann-1019003001.md.tt2

```diff
@@ -10,16 +10,23 @@ OpenResty web platform based on NGINX and LuaJIT.
 
 The (portable) source code distribution, the Win32/Win64 binary
 distributions, and the pre-built binary Linux packages for Ubuntu,
-Debian, Fedora, CentOS, RHEL, Alpine, Amazon Linux are provided on this
-[Download page](download.html).
+Debian, Fedora, CentOS, RHEL, Alpine, OpenSUSE, Amazon Linux are provided
+on this [Download page](download.html).
 
 # Version Highlights
 
-A brife overview of the most noteable changes:
+On the i9-9900K test environment, with the simple Lua Hello world interface,
+we can increase the QPS of a single worker from 221,939 rps to 269,423 rps,
+with a 20%+ improvement.
+
+And here's a brife overview of the most noteable changes:
 
 * upgraded the nginx core to 1.19.3.
-* optimize: post zombie thread: avoided the linear search.
-* feature: now we recycle lua thread GC objects for our "light threads".
+* upgraded the LuaJIT, synced code from the Mike Pall maintained branch.
```

Comment:
"synced" is informal and also incorrect. We should use the proper English word "synchronized".

"Mike Pall maintained branch" should be "the official LuaJIT repository". They are official.